# CheckRooms.action
int32 num_checks
---
# Result message
int32[] rules_broken_count
---
# Feedback message
geometry_msgs/Point robot_position
int32 rule_broken
